src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,20 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int manufactureYear;
```

Comment:
1. Consider renaming `manufactureYear` to `manufacturingYear` for better clarity 📝
2. Properties of sub-classes should be different, otherwise it may be simply moved to Machine class